client/lib/plans/constants.js

```diff
@@ -0,0 +1,23 @@
+export const plansList = {
+	free: {},
+	premium: {},
+	personal: {},
+	business: {}
+};
+
+export const featuresList = {
+	'google-analytics': {
+		title: 'Google Analytics Integration',
+		plans: [ 'premium', 'business' ]
+	},
+	'extra-storage': {
+		title: 'More storage for your media',
+		plans: [ 'personal', 'premium', 'business' ]
```

Comment:
For example how much storage space each plan will give us. These constants should eventually be populated from an api endpoint.